### DISCUSSION TITLE
File permissions of /media folder

### DISCUSSION THREAD

**Discussion by nt7** [TandoorRecipes/recipes#320] [Q&A]
Hi,

First off, thanks for the great app! 
Currently testing it to see if that's what I want to use to sort my recipes going forward.

I ran into a minor issue:
The images of my recipes are not shown, which is typically related to a file permissions error.
I'm running the app via its Docker image.
What's the recommended permission for the /media folder in Linux?

Thanks!

**Comment by vabene1111** [comment]
I suspect that this issue is related to the serving of those media files. Are the files created inside the volume ? 

If yo please post your docker-compose and your logs so i can see whats wrong.

Please also refer to this section of the documentation ragarding media serving https://vabene1111.github.io/recipes/install/docker/#nginx-vs-gunicorn
